### PR TITLE
Fixed #24584 -- Fixed MySQL microsecond handling with older MySQLdb driver

### PR DESCRIPTION
**Pull request by jdufresne**

### CODE REVIEW

File: tests/basic/tests.py [hunk 1]
```diff
@@ -202,6 +202,17 @@ def test_microsecond_precision_not_supported(self):
         self.assertEqual(Article.objects.get(id__exact=a9.id).pub_date,
             datetime(2005, 7, 31, 12, 30, 45))
 
+    @skipIfDBFeature('supports_microsecond_precision')
+    def test_microsecond_precision_not_supported_edge_case(self):
+        # In MySQL, microsecond-level precision isn't available. You'll lose
```
Comment (timgraham) [inline tests/basic/tests.py]: isn't always available?

File: tests/basic/tests.py [hunk 2]
```diff
@@ -202,6 +202,17 @@ def test_microsecond_precision_not_supported(self):
...
+        # microsecond-level precision once the data is saved.
+        a = Article.objects.create(
+            headline='Article 9',
+            pub_date=datetime(2008, 12, 31, 23, 59, 59, 999999))
```
Comment (timgraham) [inline tests/basic/tests.py]: The usual style is to put the closing parenthesis on the next line and include a trailing comma on this line.

Comment (jdufresne) [inline tests/basic/tests.py]: I was using a PEP8 style for long lines. But sure.

**Comment by claudep** [comment]
I moved the release note from 1.9 to 1.8.1 and pushed the patch, thanks!

**Pull request closed without merging**